modules/customizing-the-appearance/proc-overriding-translations.adoc

```diff
@@ -0,0 +1,107 @@
+.`allTranslations.json` fragment with translation string overrides
+[source,json]
+----
+{
+  "plugin.global-floating-action-button": {
+    "fr": {
+      "fab.quay.label": "QUAY French JSON",
+      "fab.quay.tooltip": "QUAY french tooltip JSON",
+      "fab.rbac.label": "RBAC French JSON",
+      "fab.rbac.tooltip": "RBAC french tooltip JSON"
+    },
+    "en": {
+      "fab.quay.label": "QUAY EN JSON",
+      "fab.rbac.label": "RBAC EN JSON",
+      "fab.rbac.tooltip": "RBAC EN tooltip JSON"
+    }
+  },
+  "plugin.global-header": {
+    "fr": {
+      "applicationLauncher.developerHub": "Developer Hub French JSON"
+    },
+    "en": {
+      "applicationLauncher.developerHub": "Developer Hub EN JSON"
+    }
+  }
+
+}
+----
+. Login to your cluster and create a config map for your translations override strings:
++
+[source,bash]
+----
+oc create configmap all-translations \
+    --from-file=/<path-to>/allTranslations.json
+----
+
+. Update your deployment configuration based on your installation method:
+
+.. For an Operator-installed {product-very-short} instance, update your `Backstage` custom resource (CR).
+... In the `spec.application.extraFiles` section, add the translations custom app configuration as shown in the following example:
++
+.Backstage custom resource fragment
+[source,yaml,subs="+quotes"]
+----
+apiVersion: rhdh.redhat.com/v1alpha3
+kind: Backstage
+spec:
+  application:
+    extraFiles:
+      mountPath: /opt/app-root/src/translation
+      configMaps:
+        - name: all-translations
+----
+
+.. For a Helm-installed {product-very-short} instance, update your {product-short} `Backstage` Helm chart to mount in the {product-short} filesystem your files from the `all-translations` config map:
+
+... In the {product-short} Helm chart, go to *Root Schema* → *Backstage chart schema* → *Backstage parameters* → *Backstage container additional volume mounts*.
+
+... Select *Add Backstage container additional volume mounts* and add the following values:
+
+mountPath::
+`/opt/app-root/src/translation`
+Name::
+`all-translations`
+
+... Add the translations to the *Backstage container additional volumes* in the {product-short} Helm chart:
+
+name::
+`all-translations`
+configMap::
+defaultMode:::
+  `420`
+name:::
+`all-translations`
+
+. Update the `i18n` section to your custom {product-short} `{my-app-config-file}` configuration file to include the translation override file:
++
+[id=i18n-override]
+.`{my-app-config-file}` fragment with localization `i18n` fields
+[source,yaml,subs="+quotes"]
+----
+i18n:
+  locales: # List of supported locales. Must include `en`, otherwise the translation framework will fail to load.
+    - en
+    - fr
+  defaultLocale: en # Optional. Defaults to `en` if not specified.
+  overrides: # List of JSON translation files applied in order (last file wins).  Each file may override/add translations for one or more plugins/locales
+    - <path-to>/<overrides-1>.json
+    - <path-to>/<overrides-2>.json
```

Comment:
Since we are creating a file with a specific name `all-translations` in the above example, can we use that here as well rather than having a generic filenames?
```suggestion
    - /opt/app-root/src/translations/all-translations.json
```